lib/linguist/languages.yml

```diff
@@ -4771,6 +4771,15 @@ PEG.js:
   codemirror_mode: javascript
   codemirror_mime_type: text/javascript
   language_id: 81442128
+Phel:
+  type: programming
+  tm_scope: source.lisp
+  ace_mode: lisp
+  codemirror_mode: commonlisp
+  codemirror_mime_type: text/x-common-lisp
+  color: "#512da8"
+  extensions:
+    - ".phel"
```

Comment:
You've missed step 4 of adding a language in the CONTRIBUTING.md file as there should be a `language_id` here 😉

Reply:
Thank you so much for the review @lildude 😃